js/packages/openinference-instrumentation-llama-index/src/utils.ts

```diff
@@ -94,6 +97,23 @@ export function isRetrieverPrototype(proto: unknown): proto is BaseRetriever {
   );
 }
 
+/**
+ * Checks whether the provided prototype is an instance of `LLM`.
+ *
+ * @param {unknown} proto - The prototype to check.
+ * @returns {boolean} Whether the prototype is a `LLM`.
+ */
+export function isLLMPrototype(proto: unknown): proto is LLM {
+  return (
+    proto != null &&
```

Comment:
tiny nit: i'm biased towards "clarity" (in my opinion) at the cost of verbosity - proto is a bit of a loaded term so would probably just name this param prototype